BitwardenShared/UI/Platform/Application/Views/BitwardenTextField.swift

```diff
@@ -140,11 +147,13 @@ struct BitwardenTextField<TrailingContent: View>: View {
         accessibilityIdentifier: String? = nil,
         passwordVisibilityAccessibilityId: String? = nil,
         canViewPassword: Bool = true,
+        isPasswordAutoFocused: Bool = true,
```

Comment:
🤔 By giving this `true` value as default doesn't it automatically focus the password field on the `AddEditLoginItemView`? Or what happens if the `AddEditCustomFieldsView` has several `hidden` custom fields?

Reply:
great call out, i actually shouldn't have done this.  I had it passing in the value to `true` but defaulting to false but at the last minute thought I could be clever.  Thanks for the catch!